# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../mono-develop/mono-develop-2.4.2-no-update-icon-cache.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- monodevelop-2.4.2/theme-icons/icon-theme-installer.orig	2011-05-23 11:35:34.824003157 +0200
+++ monodevelop-2.4.2/theme-icons/icon-theme-installer	2011-05-23 11:40:38.436002471 +0200
@@ -164,16 +164,12 @@
 	gtk_update_icon_cache_bin="$((which gtk-update-icon-cache || echo /opt/gnome/bin/gtk-update-icon-cache)2>/dev/null)"
 	gtk_update_icon_cache="$gtk_update_icon_cache_bin -f -t $INSTALL_BASE_DIR"
 
-	if test -z "$INSTALL_DEST_DIR"; then 
-		if test -x $gtk_update_icon_cache_bin; then 
-			echo "Updating GTK icon cache"
-			$gtk_update_icon_cache
-		else
-			echo "*** Icon cache not updated. Could not execute $gtk_update_icon_cache_bin"
-		fi
-	else
-		echo "*** Icon cache not updated. After install, run this:"
-		echo "***   $gtk_update_icon_cache"
+	if test -x $gtk_update_icon_cache_bin; then 
+		gtk_update_icon_cache_bin="gtk-update-icon-cache"
+		echo "*** Could not find $gtk_update_icon_cache_bin"
 	fi
+
+	echo "*** Icon cache not updated. After install, run this:"
+	echo "***   $gtk_update_icon_cache"
 fi
 
